### PR TITLE
Home & about pages

### PR DESCRIPTION
**Pull request by mahammad-mostafa**
- Mobile first design approach
- Responsive design on all screens
- Icons in webfont format for sharp visuals
- Hover effect on every link and icon
- Toggling hamburger mobile menu
- Dynamic loading of speakers section in home page
- Collapsing and expanding speakers list in mobile view
- Header background toggle in mobile view when scrolling
- Check loom video [here](https://www.loom.com/share/7f48ff3133ee408e9e0eadaef7176932?sid=fb3f639e-0d29-454d-b1d7-8764f453f2bb)

### CODE REVIEW

File: assets/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    description: 'Wes has worked in games for 12 years, trying to find ways to make it easier for people to translate their creative visions into reality.',
+  },
+  {
+    image: 'assets/images/speakers/zev.jpg',
+    name: 'Zev Solomon',
+    company: 'Sony Interactive Entertainment',
+    description: 'Zev has been supporting developers launching their games and integrating PlayStation Network services from PS3 to PS5.',
+  },
+  {
+    image: 'assets/images/speakers/ian.jpg',
```
Comment (Alusp) [inline assets/scripts/main.js:21]: - [ ] Good job so far, :clap: However, your image in this line of code is not showing, I think you made a mistake with your spelling: `Ian`.

possible solution: `'image: 'assets/images/speakers/Ian.jpg'`

File: assets/styles/base.css [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+  width: 40px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer {
+  padding: 50px 0;
+}
```
Comment (Alusp) [inline assets/styles/base.css:186]: - [ ] Good job on the Footer section, however, please ensure that your alignment matches the given design. a possible solution is to give `display:flex` properties and adjust them according to the design requirement. 

Below is an illustrations of yours and the given footer Design: :arrow_down: 

<table>
    <tr>
      <td>Your Footer Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

Student design footer


</td>
      <td>

design footer


</td>
    </tr>
  </table>

<table>
    <tr>
      <td>Your About Footer Design</td>
      <td>Given About Footer Design</td>
    </tr>
    <tr>
      <td>

Student about Footer

</td>
      <td>

Design about Footer


</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @mahammad-mostafa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
close

## Highlights
- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 
- [ ] No Linter Errors :green_circle: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @mahammad-mostafa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**